src/ts/spatialnavigation/FocusableContainer.ts

```diff
@@ -10,7 +10,7 @@ import { isFocusable } from './TypeGuards';
  */
 export class FocusableContainer {
   readonly container: AnyContainer;
-  readonly primaryComponent: AnyComponent;
+  readonly _primaryComponent: AnyComponent;
```

Comment:
Typescript supports `private`, what do you think about changing this field into using that?

```suggestion
  private readonly primaryComponentOverride?: AnyComponent;
```

Reply:
[3236db5](https://github.com/bitmovin/bitmovin-player-ui/pull/746/commits/3236db50a4660155e2a0d9033cde7a3474d40fcc)